odl/discr/lp_discr.py

```diff
@@ -316,6 +330,11 @@ def _astype(self, dtype):
                           exponent=self.exponent, interp=self.interp,
                           order=self.order)
 
+    def __copy__(self):
```

Comment:
Actually, this slipped in by accident since I don't longer need it. Do we want it?